transports/noise/src/lib.rs

```diff
@@ -185,6 +199,7 @@ where
         let session = self
             .params
             .into_builder()
+            .prologue(self.prologue.as_ref())
```

Comment:
Do I understand correctly that setting no prologue is equal to setting an empty prologue?

Reply:
Yes, that is my understanding. The prologue is hashed into the protocol. Appending an empty slice vs appending nothing is not going to change the final hash.